packages/http-client-java/generator/http-client-generator-test/src/test/java/com/azure/resourcemanager/resources/ArmResourceTest.java

```diff
@@ -63,10 +65,8 @@ public void testTopLevelTrackedResource() {
             .actionSync(new NotificationDetails().withMessage(NOTIFICATION_DETAILS_MESSAGE).withUrgent(true));
 
         // TopLevelTrackedResources.listByResourceGroup
-        List<TopLevelTrackedResource> topLevelTrackedResourceList = manager.topLevels()
-            .listByResourceGroup(RESOURCE_GROUP_NAME)
-            .stream()
-            .collect(Collectors.toList());
+        List<TopLevelTrackedResource> topLevelTrackedResourceList
+            = manager.topLevels().listByResourceGroup(RESOURCE_GROUP_NAME).stream().collect(Collectors.toList());
```

Comment:
What's the cause of the format change?

Reply:
I think via Alan's spotless.

`mvn package` would trigger the formatting.

Reply:
I see.. I didn't include `-Ptest` in `mvn package`.